xml/admin_ceph_gateway.xml

```diff
@@ -2936,6 +2936,14 @@ rgw_override_bucket_index_max_shards = 12
 &prompt.cephuser;ceph osd pool rm default.rgw.gc default.rgw.gc --yes-i-really-really-mean-it
 &prompt.cephuser;ceph osd pool rm default.rgw.log default.rgw.log --yes-i-really-really-mean-it
 &prompt.cephuser;ceph osd pool rm default.rgw.meta default.rgw.meta --yes-i-really-really-mean-it</screen>
+   <warning>
+     <para>
+       If you delete the default &zgroup;, you are also deleting the system user.
+       If your admin user keys are not propagated, the &dashboard;
+       will fail. Follow on to the next section to re-create your system user if
```

Comment:
I'd probably be more precise here: "the **object gateway management functionality** of the Dashboard will fail."